docs/inspec_and_friends.md

```diff
@@ -40,6 +40,14 @@ control "sshd-11" do
 end
 ```
 
+That said, InSpec is not RSpec. InSpec does not support arbitrary RSpec features. Some key differences:
+
+ * RSpec `describe` blocks are intended to be nested. In InSpec, `describe` 
+ blocks should not be nested; instead use `control` blocks to describe a 
+ higher-level grouping of tests.
+ * RSpec is a tool designed for software engineers. It thus supports a very large range of matchers, to enable testing of software engineering constructs such as exceptions, Object Oriented Programming relationships, and so on. In contrast, InSpec is aimed at compliance practioners and infrastructure testers, so our focus is providing a few, well-supported, easy-to-use universal matchers, such as `cmp`.
+ * RSpec is a separate project from InSpec. Its release schedule and feature set are beyond the control of the InSpec team. While you are free to use RSpec-specific features in your InSpec profiles, the InSpec team can only support those features described at [docs.inspec.io](https://docs.inspec.io).
```

Comment:
We should discourage the use or RSpec-specific features

Reply:
A strong discouragement of RSpec-specific features will likely alienate some members of the community. I don't know that there's any technical or logistical reason to flat-out discourage it... it's more philosophical.

I might change the wording on this to a middle-ground, such as:

> While InSpec uses parts of the RSpec project and codebase, it is a separate project from InSpec. Its release schedule and feature set are beyond the control of the InSpec team. While it is possible to use many of the RSpec core features within InSpec profiles, InSpec can only guarantee that the features described at [docs.inspec.io](https://docs.inspec.io) will function correctly. Some RSpec core functionality may be removed in future versions of InSpec as needed to ensure stability in the InSpec project.

I think this sets a better "you're on your own, we may accidentally screw you if you use RSpec core features" tone without flat-out discouraging it.